hardhat.config.ts

```diff
@@ -123,7 +123,7 @@ const config: HardhatUserConfig = {
       gas: 12000000,
       gasPrice: 'auto',
       blockGasLimit: 12000000,
-      accounts: ETH_PK ? [{ privateKey: ETH_PK, balance: (10n ** 36n).toString() }] : { mnemonic: MNEMONIC },
+      accounts: { mnemonic: MNEMONIC },
```

Comment:
Hm, but this was broken bc it means you can't use `ETH_PK` consistently. Not sure why this broke since the PR it was merged

Reply:
when we run in CI, we probably don't run with `ETH_PK` set. so it would keep using `MNEMONIC`

but when you run it locally with `ETH_PK` it will fail